sdk/test-utils/recorder/package.json

```diff
@@ -15,20 +14,20 @@
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:browser": "echo skipped",
-    "build:node": "tsc -p . && dev-tool run bundle",
+    "build:node": "tshy && npm run build:test",
     "build:samples": "echo Skipped.",
-    "build:test": "tsc -p . && dev-tool run bundle",
-    "build": "npm run clean && tsc -p . && dev-tool run bundle",
+    "build:test": "tshy && dev-tool run build-test ",
+    "build": "npm run clean && npm run build:test",
     "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
     "extract-api": "echo skipped",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "concurrently \"npm run tests:server\" \"npm run test:browser-with-proxy\" --kill-others --success first",
     "integration-test:node": "concurrently \"npm run tests:server\" \"npm run test:node-with-proxy\" --kill-others --success first",
-    "test:node-with-proxy": "dev-tool run test:node-ts-input -- --timeout 1200000 \"test/*.spec.ts\"",
-    "test:browser-with-proxy": "dev-tool run test:browser",
+    "test:node-with-proxy": "dev-tool run test:vitest -- \"test/*.spec.ts\"",
+    "test:browser-with-proxy": "dev-tool run test:vitest --browser",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "tests:server": "cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\": \\\"commonjs\\\"}\" ts-node test/utils/server.ts",
+    "tests:server": "ts-node --esm server.ts",
```

Comment:
Why not just use `server.mjs` at the root instead?  There's no reason we should be needing to use a TypeScript file for this, and just use `node server.mjs`

Reply:
Also make sure in `server.mjs` uses `node:` prefixes for Node built-ints